### PR TITLE
Improve the doc of k8s-keystone-auth

### PR DESCRIPTION
**Pull request by openstacker**
**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
```

### CODE REVIEW

File: docs/using-keystone-webhook-authenticator-and-authorizer.md [hunk 1]
```diff
@@ -64,9 +64,49 @@ This command creates the following entry in your ~/.kube/config
     auth-provider:
       name: openstack
 ```
-- Run `kubectl config set-context --cluster=kubernetes --user=openstackuser openstackuser@kubernetes`
+- Run `kubectl config set-context --cluster=<your cluster name> --user=openstackuser openstackuser@kubernetes`
```
Comment (gonzolino) [inline docs/using-keystone-webhook-authenticator-and-authorizer.md]: In this command you use "<your cluster name>" as cluster name, in the kubeconfig below you use "mycluster".
It's just a personal opinion, but I would prefer to use the same name in both examples.

Comment (openstacker) [reply docs/using-keystone-webhook-authenticator-and-authorizer.md]: @gonzolino Thanks for the comments, do you mean using --cluster=mycluster at here?

Comment (gonzolino) [reply docs/using-keystone-webhook-authenticator-and-authorizer.md]: Yes that would be my suggestion. Or, alternatively, you could use "<your cluster name>" in the kubeconfig example.
Either way is ok for me, but I think it should be consistent.

Comment (openstacker) [reply docs/using-keystone-webhook-authenticator-and-authorizer.md]: Ok, cool. I will do that. The current '--cluster=kuberntes' is very confusing, hence why I'd like to update it. Thanks. Will propose a new patch set.

**Pull request merged**